### PR TITLE
Update old function, remove mobile robot wrapper

### PR DESCRIPTION
**Pull request by kozakromch**
Hello. I try to do this exercise, but noticed that there are a lot of deprecated or not working function. 
class mobileRobotWrapper not working, and I don't know how to fix it, so I remove it.

### CODE REVIEW

**Comment by jcarpent** [comment]
Thanks for the contribution. 
The ambition of having the mobile platform is to solve more complex IK problems as you might have noticed.
What is not working in particular?
@proyan Could you help on this task?

**Comment by kozakromch** [comment]
Oh. Okay, I get it. In MobileRobot instead of call super.__init__ I should call initFromUrdf. 

But actually I don't know the final result. Is it okay that robot could slide away from mobile platform ?
image

**Comment by proyan** [comment]
I'm on holiday today and tomorrow, I'll be able to look at it Monday

File: doc/d-practical-exercises/3-invkine.md [hunk 1]
```diff
@@ -21,28 +21,35 @@ the tool. They are displayed when the robot moves.
...
-from mobilerobot import MobileRobotWrapper
+import pinocchio as pin
 from pinocchio.utils import *
-
-PKG = '/opt/openrobots/share'
-URDF = join(PKG, 'ur5_description/urdf/ur5_gripper.urdf')
-
-robot = MobileRobotWrapper(URDF, [PKG])
+from os.path import dirname, join, abspath
+from mobilerobot import MobileRobotWrapper
```
Comment (proyan) [inline doc/d-practical-exercises/3-invkine.md:27]: I thought the purpose of this PR was to remove mobilerobot? :)
Could you instead use robotwrapper here?

File: doc/d-practical-exercises/3-invkine.md [hunk 2]
```diff
@@ -21,28 +21,35 @@ the tool. They are displayed when the robot moves.
...
 
-# robot.viewer.gui.addFloor('world/floor')
-
 NQ, NV = robot.model.nq, robot.model.nv
 
-q = rand(NQ)
-robot.display(q)
+# create valid random position
+q = pin.integrate(robot.model, pin.neutral(robot.model), np.random.rand(NV) * 10)
+robot.viz.display(q)
```
Comment (proyan) [inline doc/d-practical-exercises/3-invkine.md:47]: just robot.display should be fine, you don't need robot.viz here, or in other places in this PR.

File: doc/d-practical-exercises/3-invkine.md [hunk 3]
```diff
@@ -21,28 +21,35 @@ the tool. They are displayed when the robot moves.
...
 robot.initDisplay(loadModel=True)
 
-# robot.viewer.gui.addFloor('world/floor')
-
 NQ, NV = robot.model.nq, robot.model.nv
 
-q = rand(NQ)
-robot.display(q)
+# create valid random position
+q = pin.integrate(robot.model, pin.neutral(robot.model), np.random.rand(NV) * 10)
```
Comment (proyan) [inline doc/d-practical-exercises/3-invkine.md:46]: you should use pin.randomConfiguration here, no need to use this trick to handle lie spaces

File: doc/d-practical-exercises/3-invkine.md [hunk 4]
```diff
@@ -54,53 +61,55 @@ goal placement.
 
 ```py
 def place(name, M):
-    robot.viewer.gui.applyConfiguration(name, se3ToXYZQUAT(M))
-    robot.viewer.gui.refresh()
+    robot.viz.viewer.gui.applyConfiguration(name, pin.SE3ToXYZQUAT(M).tolist())
```
Comment (proyan) [inline doc/d-practical-exercises/3-invkine.md:64]: same here, robot.viewer is directly available, and we don't need viz here

**Review by proyan — APPROVED** [review]

File: doc/d-practical-exercises/3-invkine.md [hunk 5]
```diff
@@ -21,28 +21,35 @@ the tool. They are displayed when the robot moves.
...
-q = rand(NQ)
+# create valid random position
+q = pin.randomConfiguration(robot.model)
 robot.display(q)
 
 IDX_TOOL = 24
 IDX_BASIS = 23
 
-se3.framesKinematics(robot.model, robot.data)
+pin.updateFramePlacements(robot.model, robot.data)
```
Comment (proyan) [inline doc/d-practical-exercises/3-invkine.md:52]:
```suggestion
pin.framesForwardKinematics(robot.model, robot.data, q)
```

File: doc/d-practical-exercises/3-invkine.md [hunk 6]
```diff
@@ -54,53 +61,56 @@ goal placement.
...
-    q = se3.Quaternion(x, y, z, w)
+    q = pin.Quaternion(x, y, z, w)
     q.normalize()
     return q.matrix()
 
-Mgoal = se3.SE3(Rquat(.4, .02, -.5, .7), np.matrix([.2, -.4, .7]).T)
-robot.viewer.gui.addXYZaxis('world/framegoal', [1., 0., 0., 1.], .015, 4)
+Mgoal = pin.SE3(Rquat(.4, .02, -.5, .7), np.matrix([.2, -.4, .7]).T)
+robot.
+viewer.gui.addXYZaxis('world/framegoal', [1., 0., 0., 1.], .015, 4)
```
Comment (proyan) [inline doc/d-practical-exercises/3-invkine.md:74]:
```suggestion
robot.viewer.gui.addXYZaxis('world/framegoal', [1., 0., 0., 1.], .015, 4)
```

**Review by proyan — CHANGES_REQUESTED** [review]

**Review by proyan — APPROVED** [review]

**Comment by proyan** [comment]
Thanks @kozakromch for the contribution.

**Review by jcarpent — APPROVED** [review]
Thanks @proyan and @kozakromch for handling this PR.

**Pull request merged**